.github/workflows/build.yml

```diff
@@ -2,7 +2,28 @@ name: 🛠️ Build + Test
 
 on:
   push:
+    paths:
+      - '*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}'
+      - 'src/core/icons/{file,folder,language}Icons.ts'
+      - 'icons/*.svg'
+      - 'build/'
+      - 'src/core'
+      - 'src/extension'
+      - 'src/module'
+      - 'bunfig.toml'
+      - 'package.json'
+
   pull_request:
+    paths:
+      - '*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}'
+      - 'src/core/icons/{file,folder,language}Icons.ts'
```

Comment:
Isn't that redundant?

- 'src/core/icons/{file,folder,language}Icons.ts'
- 'src/core'